Document undocumented event (#1472)
Pycord-Development/pycord: docs/api.rst

    :param exception: The DiscordException associated to the error.
    :type exception: :class:`DiscordException`
    
.. function:: on_unknown_application_command(interaction)

    Called when an application command is not found in bot's internal cache.

    .. versionadded:: 2.0

>     :param interaction: The interaction associated to the command that is not found.

Reviewer: ```suggestion
    :param interaction: The interaction associated to the unknown command.
```
